### PR TITLE
Support for ARM64 platform is added to the project

### PR DESCRIPTION
**Pull request by svok**
Please, accept this pull request. It adds support for LinuxArm64 platform

### CODE REVIEW

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -5,10 +5,10 @@ binaryCompabilityValidator = "0.13.2"
 toolchainsResolver = "0.5.0"
 detekt = "1.23.1"
 mavenPublish = "0.25.3"
-coroutines = "1.7.2"
-atomicfu = "0.21.0"
+coroutines = "1.7.3"
+atomicfu = "0.22.0"
 lincheck = "2.17"
-statelyIso = "1.2.5"
+statelyIso = "2.0.4"
```
[the remark this answers is not in the document]
Comment (svok) [reply gradle/libs.versions.toml]: I've rolled back unnecessary versions update. statelyIso = "2.0.4" is required. statelyIso = "1.2.5" doesn't support arm64.

**Comment by ychescale9** [comment]
Thanks!

**Comment by vdshb** [comment]
@ychescale9 Any plans to publish a release with this architecture?

**Comment by ychescale9** [comment]
Just released a new version 😃

**Pull request merged**